core/src/application/Game.java

```diff
@@ -5,6 +5,9 @@
 import view.View;
 import com.badlogic.gdx.ApplicationAdapter;
 
+/**
+ * Runs the main loop of the playable game, in other words the ingame loop
```

Comment:
Looks like a description of a method.

Reply:
Application the create and connects medel, view, controller